### PR TITLE
Feat : Added Responsive Image

### PR DESCRIPTION
**Pull request by srejitk**
### Added Responsive Image

This feature push will add the responsive Image component. 

They're are 3 variants 

1. Round
2. Square
3. Squircle
Images

### CODE REVIEW

**Review by nishasen — COMMENTED** [review]
You can reduce the extra space, otherwise everything looks good

File: stylesheets/components/Image/images.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  
+      .image--round img {
+        border-radius: 50%;
+        width: 200px;
+        height: 200px;
+      }
+  
+      .image--square img {
+        width: 200px;
+        height: 200px;
```
Comment (nakulsharma15) [inline stylesheets/components/Image/images.css]: same as above

File: stylesheets/components/Image/images.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      }
+  
+      .image--squircle img {
+        width: 200px;
+        height: 200px;
+        border-radius: 28px;
```
Comment (nakulsharma15) [inline stylesheets/components/Image/images.css]: same as above

File: stylesheets/components/Image/images.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+    /* Responsive Image */
+    .image--responsive {
+        max-width: 100%;
+        object-fit: cover;
+      }
+  
+      .image--round img {
+        border-radius: 50%;
+        width: 200px;
+        height: 200px;
```
Comment (nakulsharma15) [inline stylesheets/components/Image/images.css]: you can use rem here

**Review by nakulsharma15 — COMMENTED** [review]
suggested some minor changes

File: stylesheets/components/Image/images.css (hunk 1 above)
Comment (srejitk) [reply stylesheets/components/Image/images.css]: Thank You Nakul. Corrected in the latest commit.

File: stylesheets/components/Image/images.css (hunk 2 above)
Comment (srejitk) [reply stylesheets/components/Image/images.css]: Corrected in the latest commit.

File: stylesheets/components/Image/images.css (hunk 3 above)
Comment (srejitk) [reply stylesheets/components/Image/images.css]: Corrected in the latest commit.

**Pull request merged**